internal/armada/repository/event.go

```diff
@@ -2,29 +2,27 @@ package repository
 
 import (
```

Comment:
The diffs here are quite confusing.  This is the new event repository that loads up the stored messages from pulsar.  The previous repository has moved to `legacy_event.go`